android-auto-app/src/main/AndroidManifest.xml

```diff
@@ -17,10 +16,6 @@
         android:supportsRtl="true"
         android:extractNativeLibs="false">
 
-        <meta-data
-            android:name="androidx.car.app.minCarApiLevel"
-            android:value="1"/>
-
```

Comment:
Moving this back here. This will let downstream developers decide what their minCarApiLevel is

Reply:
oh never mind it is already like this, this is just a duplicate in the current `main` manifest